tsl/test/expected/bgw_job_stat_history_errors.out

```diff
@@ -30,15 +30,14 @@ CREATE OR REPLACE PROCEDURE custom_proc1(jobid int, config jsonb) LANGUAGE PLPGS
 $$
 BEGIN
   UPDATE custom_log set msg = 'msg1' where msg = 'msg0';
-  perform pg_sleep(10);
+  perform pg_sleep(5);
   COMMIT;
 END
 $$;
 CREATE OR REPLACE PROCEDURE custom_proc2(jobid int, config jsonb) LANGUAGE PLPGSQL AS
 $$
 BEGIN
   UPDATE custom_log set msg = 'msg2' where msg = 'msg0';
-  perform pg_sleep(10);
```

Comment:
It doesn't look like `custom_proc2` waits for anything in this test so I eliminated this sleep call entirely.

Reply:
The problem here is actually such kind of test should be an isolation test instead. This sleep is to make sure we'll end up with a concurrent update issue: https://github.com/timescale/timescaledb/blob/main/tsl/test/expected/bgw_job_stat_history_errors.out#L96